src/views/components/common/trace-detail-statistics-table.vue

```diff
@@ -0,0 +1,351 @@
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+<template>
+  <div class="trace-detail-chart-table">
+    <div class="rk-trace-t-loading" v-show="loading">
+      <svg class="icon loading">
+        <use xlink:href="#spinner"></use>
+      </svg>
+    </div>
+    <TraceContainer :tableData="tableData" :type="HeaderType">
+      <div class="trace-tips" v-if="!tableData.length">{{ $t('noData') }}</div>
+    </TraceContainer>
+    <rk-sidebox :width="'50%'" :show.sync="showDetail" :title="$t('spanInfo')">
+      <TraceSpanLogs :currentSpan="currentSpan" />
+    </rk-sidebox>
+  </div>
+</template>
+<style lang="scss">
+  .rk-tooltip-popper.trace-table-tooltip .rk-tooltip-inner {
+    max-width: 600px;
+  }
+  .trace-detail-chart-table {
+    position: relative;
+    min-height: 300px;
+    border-bottom: 1px solid #ccc;
+  }
+</style>
+
+<script lang="js">
+  import copy from '@/utils/copy';
+  import TraceContainer from './trace-statistics-table/trace-container';
+  import _ from 'lodash';
+  import TraceSpanLogs from '../trace/trace-span-logs.vue';
+  /* eslint-disable */
+  /* tslint:disable */
+  export default {
+    components: {
+      TraceContainer,
+      TraceSpanLogs,
+    },
+    props: ['data', 'traceId', 'showBtnDetail', 'HeaderType'],
+    watch: {
+      data(val, oldVal) {
+        if (!this.data.length) {
+          this.tableData = [];
+          return;
+        }
+        this.tableData = this.formatData(this.changeTree());
+        this.loading = false;
+      },
+    },
+    data() {
+      return {
+        tableData: [],
+        diaplay: true,
+        // segmentId: [],
+        showDetail: false,
+        list: [],
+        currentSpan: [],
+        loading: true,
+      };
+    },
+    methods: {
+      copy,
+      // TODO 统计计算
```

Comment:
Same issue, and why TODO?

Reply:
sorry,to remind my partner,and he forget to remove this exegesis,I'll remove it and commit

Reply:
Please confirm with your colleague, if this PR gets merged, his name and mail wouldn't show up on the ASF. 
It is better you could ask him to confirm here, publicly. Usually, a pull request should belong to one individual.

Reply:
I think his GitHub ID is @Visbin